modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -130,6 +130,39 @@ class Test_ONNX_layers : public DNNTestLayer
         if (checkNoFallbacks)
             expectNoFallbacksFromIE(net);
     }
+
+    void testONNXModelRead(const String& basename, const Extension ext = npy,
```

Comment:
Do we really need to test read? It should also work without the support of boolean.

Reply:
Actually it was failing initially, that is why I added this patch. Test was attended due to @dkurt suggestion.

Reply:
dnn cannot deal with boolean internal blobs for now. Do you plan to make subsequent PRs to support this? I guess we need to have type inference first to make this happen though.

Reply:
Exactly, that was our idea to fix ONNX import first and then wait for type inference.